pkg/database/user_report.go

```diff
@@ -108,6 +108,22 @@ func (db *Database) FindUserReport(tx *gorm.DB, phoneNumber string) (*UserReport
 	return &ur, nil
 }
 
+// DeleteUserReport removes a specific phone number from the user report
+// de-duplication table.
+func (db *Database) DeleteUserReport(phoneNumber string) error {
+	return db.db.Transaction(func(tx *gorm.DB) error {
+		ur, err := db.FindUserReport(tx, phoneNumber)
```

Comment:
Why do we need to lookup before deleting? It feels like this function is declarative (after I call it, the UR doesn't exist). Do we need to process the situation in which the record doesn't exist?

```go
db.db.Where("phone = ?", phoneNumber).Delete(&UserReport{})
```

Reply:
true - it also saves a transaction.